Format graphic_matroid.py

gmou3 opened the pull request:
<!-- ^^^^^
Please provide a concise, informative and self-explanatory title.
Don't put issue numbers in there, do this in the PR body below.
For example, instead of "Fixes #1234" use "Introduce new method to calculate 1+1"
-->
<!-- Describe your changes here in detail -->
Improve the formatting of graphic_matroid.py.

gmou3 committed:
Format graphic_matroid.py

gmou3 committed:
Merge remote-tracking branch 'sagemath/develop' into lint_graphic_matroid

tscrim commented on src/sage/matroids/graphic_matroid.py:
```diff
@@ -1357,26 +1397,29 @@ def graphic_extension(self, u, v=None, element=None):
             sage: list(M1.graph().edge_iterator())
-            [(0, 1, 'a'), (0, 1, 0), (0, 2, 1), (0, 3, 2), (1, 2, 3), (1, 3, 4), (2, 3, 5)]
+            [(0, 1, 'a'), (0, 1, 0), (0, 2, 1), (0, 3, 2), (1, 2, 3),
+            (1, 3, 4), (2, 3, 5)]
```
```suggestion
            [(0, 1, 'a'), (0, 1, 0), (0, 2, 1), (0, 3, 2), (1, 2, 3),
             (1, 3, 4), (2, 3, 5)]
```

tscrim commented on src/sage/matroids/graphic_matroid.py:
```diff
@@ -1357,26 +1397,29 @@ def graphic_extension(self, u, v=None, element=None):
+            sage: sorted(EX_G.vertex_iterator(), key=str)
             [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'b']
-            sage: M.graphic_extension('a', 'b', 'c').graph().vertices(sort=False)
+            sage: EX_G = M.graphic_extension('a', 'b', 'c').graph()
```
```suggestion
            sage: M.graphic_extension('a', 'b', 'c').graph()
```
No need to suggest it is being stored.

tscrim commented on src/sage/matroids/graphic_matroid.py:
```diff
@@ -1688,19 +1746,21 @@ def graphic_coextensions(self, vertices=None, v=None, element=None, cosimple=Fal
             136
             sage: I = M.graphic_coextensions(cosimple=True)
-            sage: sum(1 for N in I)
+            sage: len(list(I))
```
These actually put everything in memory rather than iterating over it. So it is actually a slightly different doctest. I think the previous behavior is a bit better to suggest to the user.

tscrim reviewed (commented):
Thank you for working to clean up the file. However, most of your changes actually make the file harder to understand the code and output. The 80 char/line can often be bad to follow rigidly, which can be seen by line splits with small output for just a few characters over. While we generally follow this more strictly for docstrings, there are good reasons to violate this at times for readability, especially when the html output doesn't require a horizontal scrollbar. So unfortunately I think most of these changes should be reverted.

Also, IMO, the strong avoidance of `l` because it could be confused with `I` thing only is an issue if both appear or there is ambiguity with what symbol it should mean. As such, calling labels `ll` instead of `l` doesn't actually fix the problem (why isn't it `II` instead?) but just makes the names longer and more confusing.

gmou3 committed:
Changes suggested by tscrim, "ground set" to "groundset", 1-line outputs

Revert some changes and enforce line limits: docstrings to 80 and example blocks to 95.

tscrim commented on src/sage/matroids/graphic_matroid.py:
```diff
@@ -544,8 +533,9 @@ def _minor(self, contractions=frozenset([]), deletions=frozenset([])):
-            sage: N = M._minor(deletions = frozenset([0, 3, 5, 9]), contractions =
-            ....: frozenset([1, 2, 11])); N
+            sage: N = M._minor(deletions = frozenset([0, 3, 5, 9]),
+            ....:              contractions = frozenset([1, 2, 11]))
```
```suggestion
            sage: N = M._minor(deletions=frozenset([0, 3, 5, 9]),
            ....:              contractions=frozenset([1, 2, 11]))
```
PEP8

tscrim reviewed (approved):
Thank you. One last little PEP8 thing since you are modifying the line.

gmou3 committed:
Small PEP8 edit

Suggested by tscrim

Co-authored-by: Travis Scrimshaw <clfrngrown@aol.com>
